app/controllers/errors_controller.rb

```diff
@@ -10,6 +10,7 @@ class ErrorsController < ApplicationController
   end
 
   def auth_error
+    @page_title = "Auth Error"
```

Comment:
I think this needs to be `@page_subtitle` so we don't lose the ` | Archive of Our Own` that appears after `Auth Error`.